lib/rubocop/plugin.rb

```diff
@@ -23,11 +23,15 @@ def plugin_capable?(feature_name)
         return true if BUILTIN_INTERNAL_PLUGINS.key?(feature_name)
         return true if feature_name == OBSOLETE_INTERNAL_AFFAIRS_PLUGIN_NAME
 
-        # When not using Bundler. Makes the spec available in loaded_specs but does not require it.
-        Gem.try_activate(feature_name)
-        return false unless (gem = Gem.loaded_specs[feature_name])
-
-        !!gem.metadata['default_lint_roller_plugin']
+        begin
+          # When not using Bundler. Makes the spec available but does not require it.
+          gem feature_name
+        rescue Gem::LoadError
+          false
```

Comment:
Aren't you missing a `return` here? I thought `LInt/Void` should have caught something like this.

Reply:
Yeah, I meant to do a return here. But, this is already covered by the next return below so I now simply removed it.

I think right now `Lint/Void` is only capable of checking nodes that appear last, so no offense here